stack88.yaml

```
@@ -1,4 +1,4 @@
-resolver: nightly-2020-02-13
+resolver: nightly-2020-03-26
 packages:
 - .
 extra-deps:
```

```suggestion
extra-deps:
- base-compat-0.11.0
- base-compat-batteries-0.11.0
```

Downgrading base-compat seems to fix the build: https://dev.azure.com/jneira/haskell-language-server/_build/results?buildId=738&view=logs&j=c161c675-1417-57ed-29bf-7d19b694489b&t=3813d704-4df5-5eff-d5f1-7414d6aa39b0
(At least it builds base-compat)

---

But then `shake` throws the error too:
```
shake                            > Access violation in generated code when writing 0x0
shake                            > 
shake                            >  Attempting to reconstruct a stack trace...
shake                            > 
shake                            >    Frame	Code address
shake                            >  * 0x461da80	0x37f7b66 C:\Users\VssAdministrator\AppData\Local\Programs\stack\x86_64-windows\ghc-8.8.3\bin\ghc.exe+0x33f7b66
shake                            >  * 0x461da88	0x3277bb9 C:\Users\VssAdministrator\AppData\Local\Programs\stack\x86_64-windows\ghc-8.8.3\bin\ghc.exe+0x2e77bb9
shake                            >  * 0x461da90	0x5
shake                            >  * 0x461da98	0x5
shake                            >  * 0x461daa0	0x3d7dc22 C:\Users\VssAdministrator\AppData\Local\Programs\stack\x86_64-windows\ghc-8.8.3\bin\ghc.exe+0x397dc22
shake                            >  * 0x461daa8	0x5dc0d40
shake                            > 
```
😞

---

Hmm, why is this PR failing in a similar way and is only changing the `README`?

https://github.com/digital-asset/ghcide/pull/513/checks?check_run_id=541282737

https://dev.azure.com/digitalasset/ghcide/_build/results?buildId=37435&view=logs&j=c161c675-1417-57ed-29bf-7d19b694489b&t=f76bb9b6-f207-5559-7acf-4fd4ad912d7e&l=508

```Access violation in generated code when reading ffffffffffffffff```

It's with resolver: nightly-2020-02-13

https://github.com/ndmitchell/ghcide/blob/b75de335653466cd05e80f33c57e1daace4e86f3/stack88.yaml#L1

Just want to make sure `8.8.3` is the cause and not Windows Azure CI.

---

Well, maybe both are guilty 😄 
* It works in my windows 10, even after wiping out stack root and `.stack-work`
* But the error is fized for a library changing its version (and it is terribly similar to the other ones in the ticket)

Maybe we could give a try to a previous windows version in azure-ci, but i would not like to left it as default

---

And imo the error in that build  is not related:
```
Access violation in generated code when reading ffffffffffffffff
```

---

"Access violation" was the only similarity. Everything else is different, though.